src/ewig/keys.cpp

```diff
@@ -69,7 +70,7 @@ namespace {
 
 key_seq from_special_str(const char* name)
 {
-    auto id = ::tigetstr(name);
+    auto id = ::tigetstr((NCURSES_CONST char*)name);
```

Comment:
Is this removing constnes?

Reply:
Yes, NCURSES_CONST was defined empty in my version of ncurses.

Reply:
Funny...